### ISSUE TITLE
Change green buttons to action links on how to apply and eligibility pages

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#40572]
## Issue Description

Action links were recently added to the design system to sometimes be used in places where we used to have green buttons. There are still situations where we should be using a button so before changing all pages, confirm that an action link is the proper component. [Read about when to use action links](https://design.va.gov/components/link/action)

To start, we need to update our how to apply and eligibility pages. The design council has discussed monitoring analytics to make sure there's not a huge drop in clicks when this design changes so we are not changing the action links on all pages.

Pages to change:
- [x] [How to file a VA disability claim](https://www.va.gov/disability/how-to-file-claim/)
- [ ] [How to apply for VA health care](https://www.va.gov/health-care/how-to-apply/)
- [ ] [Eligibility for VA health care](https://www.va.gov/health-care/eligibility/)
- [x] [Eligibility for Veterans Pension](https://www.va.gov/pension/eligibility/)
- [x] [How to apply for a VA pension as a Veteran](https://www.va.gov/pension/how-to-apply/)
- [x] [How to apply for Veteran Readiness and Employment](https://www.va.gov/careers-employment/vocational-rehabilitation/how-to-apply/)
- [x] [Eligibility for Veteran Readiness and Employment](https://www.va.gov/careers-employment/vocational-rehabilitation/eligibility/)
- [x] [Educational and career counseling](https://www.va.gov/careers-employment/education-and-career-counseling/)
- [x] [Edith Nourse Rogers STEM scholarship](https://www.va.gov/education/other-va-education-benefits/stem-scholarship/)
- [x] [Higher Level Reviews](https://www.va.gov/decision-reviews/higher-level-review/)
- [x] [How to request a VA home loan Certificate of Eligibility](https://www.va.gov/housing-assistance/home-loans/how-to-request-coe/)
- [x] [Eligibility requirements for VA home loan programs](https://www.va.gov/housing-assistance/home-loans/eligibility/)
- [x] [How to apply for a Specially Adapted Housing grant](https://www.va.gov/housing-assistance/disability-housing-grants/how-to-apply/)

We will create a separate ticket for the backlog to change other pages once we review the analytics on these pages.

## Tasks
- [x] Read style guide guidance for when to use action links and confirm (Aliyah)
- [x] Change buttons to action links on pages listed above (Aliyah) 
- [x] Review with pre-publishing checklist (Laura)
- [x] Publish pages (Laura)
- [ ] Create ticket to change buttons on other pages (Randi)

## Acceptance Criteria
- [x] All pages listed above are now live with action links

**Comment by RLHecht** [comment]
@aliyahblackmore @megzehn This one will have to wait until you're both set up in Drupal so nothing to do for now.

**Comment by RLHecht** [comment]
@aliyahblackmore I've updated this ticket to show that we are only changing the pages above for now. Please still review the button/action link guidance in the style guide to confirm that these pages all should be changed to action links.

**Comment by aliyahblackmore** [comment]
Noting here that I have reviewed all of the above pages and each will need an action link (based on the[ criteria](https://design.va.gov/components/link/action)).  I will input action links on Drupal on Monday! cc: @RLHecht

**Comment by aliyahblackmore** [comment]
Hi @laurwill - passing this to you. All of the "checked" links have been updated with action links.  Noting here that the links that do not have a "check" have react widgets and @RLHecht will be in touch with Beth and Danielle about next steps on those. Randi gave the okay to review/publish the pages that are complete.

**Comment by aliyahblackmore** [comment]
Noting here that Randi confirmed that I should go ahead and remove react widgets where necessary. I will work on this today, May 18!

**Comment by laurwill** [comment]
Thanks Aliyah! Noting that I reviewed and published all the checked pages — I'll plan to review and publish the pages with react widgets once those are done.

Also noting that in addition to the primary action link on the [How to file a VA disability claim](https://www.va.gov/disability/how-to-file-claim/) page, we changed the secondary blue "Check the status of your claim" button to a secondary action link.

**Comment by RLHecht** [comment]
@DanielleThierryUSDSVA Did you confirm with the health care teams that it's OK for us to remove the react widgets off the page so we can put in action links?

**Comment by DanielleThierryUSDSVA** [comment]
@RLHecht. I've confirmed that we should **not** remove the react widget buttons on the health care pages.   

Authenticated visitors get their application saved in progress status there instead of the apply button.

The function that creates the widget and subsequent button is from Platforms Save in Progress codebase. To change the button to an action link, we'd have to go through Platform to have them update their widget function code. 

Could you connect with Platform on this when you have a chance?

**Comment by RLHecht** [comment]
Just realized we still need to publish the Veterans Pension eligibility page and the Educational and Career Counseling pages since these 2 were in react widgets. @megzehn , would you be able to do these? Laura had done the other pages while you were out. Also noting that for the Veterans Pension page, we need to make sure the action link says "Veterans" instead of "Veteran." I just noticed this in Drupal. Thank you!

I reached out to Platform about the health care pages and waiting to hear back from them on when they can update for us.
